graders/python/python_autograder/pl_result.py

```diff
@@ -93,7 +93,7 @@ def addError(self, test, err):
                 ipynb_key=Feedback.test.ipynb_key,
             )
         else:
-            tr_message = ''.join(traceback.format_exception(*err))
+            tr_message = "".join(traceback.format_exception(*err))
```

Comment:
This was changed when I ran `make format`. It wasn't caught previously as we don't mount `images` into the Docker container, so our Python linters never run on them.